libs/payments/cart/src/lib/cart.service.ts

```diff
@@ -244,7 +246,6 @@ export class CartService {
     const [upcomingInvoice, eligibility] = await Promise.all([
       this.invoiceManager.previewUpcoming({
         priceId: price.id,
-        currency,
```

Comment:
Isn't currency a required field for SP3? How are none default currency values determined, i.e. EUR etc?